Makefile

```diff
@@ -2,12 +2,12 @@
 
 lint:
 	@luacheck lua/lualine
-	@luacheck tests
+	@luacheck lua/tests
 
 format:
 	@for file in `find -name '*.lua'`;do lua-format $$file -i; done;
 
 test:
-	@nvim --headless -u tests/minimal_init.lua -c "PlenaryBustedDirectory tests/ { minimal_init = './tests/minimal_init.lua' }"
+	@nvim --headless -u lua/tests/minimal_init.lua -c "PlenaryBustedDirectory lua/tests/ { minimal_init = './lua/tests/minimal_init.lua' }"
```

Comment:
I'm actually against putting `tests` in `lua/` . As it unnessarily puts them on user's path .Ypi shluld already be able to do `require'lualine'` . As it's in path. If you want to require test files too apply the patch for `minimal_init` from 7fd434f76f2c2d06c03d75ea1ebe8b55eb8eec83 . I couldn't find much use of it thet's why I removed it later /

Reply:
> This is exactly the problem I'm having however this patch does not help.

`require'tests.helper'` will need to be changed to `require'helper'` .

Also maybe try puting lualine's directory (from pwd) instead of '.'

Reply:
> `require'tests.helper'` will need to be changed to `require'helper'` .

this does not help unfortunately.

> Also maybe try puting lualine's directory (from pwd) instead of '.'

lualine modules load properly, the issue is loading test files. Does this work properly on your machine with clean branch?

Reply:
>Does this work properly on your machine with clean branch?

Just cloned master on a separate directory and tests ran fine

Reply:
Do you just get error for `build_component_opts` and not for `eq` or `meths` ?

Also It should be `require'helpers'` I missed the s in the end last time.